.github/workflows/honggfuzz.yml

```
@@ -0,0 +1,84 @@
+name: Run nightly fuzzer jobs
+
+on:
+  schedule:
+    - cron: '0 12 * * *'
```

this should probably be:

0 0 * * *

---

good catch, this is a copy-paste error from my own crontab
```suggestion
    - cron: '0 0 * * *'
```